src/storybook/components/frame/frams.scss

```diff
@@ -0,0 +1,7 @@
+@import "../../../styles/global-css-settings";
+
+.monday-storybook-frame {
+  border-radius: $border-radius-small;
+  box-shadow: 0 0 3px rgba(0, 0, 0, 0.19);
```

Comment:
this as well

Reply:
different shadow in the variables. change the design?